checker/jdk/Makefile.jdk

```diff
@@ -30,8 +30,8 @@ build: classes
 
 ############## Validate annotated jdk against missing methods ##############
 
-SOURCES = $(shell find src/java src/javax -name '*.java')
-# CLASSES = $(shell find src/java src/javax -name '*.class')
+SOURCES = $(shell find src/java src/javax src/org/w3c src/org/xml -name '*.java')
```

Comment:
`src/com` doesn't seem to contain JDK classes, so I didn't include it here. I would have used `src/org`, but I wasn't sure if other non-JDK classes would be placed here as well.